tests/test_logger.py

```
@@ -0,0 +1,23 @@
+from __future__ import annotations
+
+from pathlib import Path
+import sys
+
+sys.path.insert(0, str(Path(__file__).resolve().parents[1]))
+
+from femtologging import FemtoLogger
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)